server/src/internalClusterTest/java/org/opensearch/search/SearchWeightedRoutingIT.java

```diff
@@ -804,9 +804,8 @@ public void testMultiGetWithNetworkDisruption_FailOpenDisabled() throws Exceptio
 
     /**
      * Assert that preference based search is not allowed with strict weighted shard routing
-     * @throws Exception throws exception
      */
-    public void testStrictWeightedRouting() throws Exception {
+    public void testStrictWeightedRouting() {
```

Comment:
These un-documented @throws created issues while running the tests on local. Not related to the fix of the flaky test.

Reply:
@sachinpkale Which JDK are you using locally? I've seen inconsistencies with some non-LTS versions with regard to javadoc enforcement so I'd recommend using 11 or 17 if that is the cause here.

Reply:
@andrross the javadoc enforcement should not apply over test code, do you see checks enforced somewhere?

Reply:
@reta Just from this comment: https://github.com/opensearch-project/OpenSearch/issues/5995#issuecomment-1423021005 I have no idea _why_ JDK 14 seems to want to enforce javadoc rules against tests, but that's what Tianli's comment suggests is happening.

Reply:
Oh I see, thanks @andrross